tests/test_money.py

```diff
@@ -153,3 +153,39 @@ def test_sub_negative():
     for bill in bills:
         total -= bill
     assert total == Money(-33, "EUR")
+
+
+def test_add_result_in_proper_instance():
+    one = Money(1, "EUR")
+    assert one._decimal_places_display is None, "this should be None"
+    assert one.decimal_places == 2
+    two = Money(2, "EUR")
+    assert two._decimal_places_display is None, "this should be None"
+    assert two.decimal_places == 2
+    three = one + two
+    assert three._decimal_places_display is None, "this should be None as the others are None"
+    assert three.decimal_places == 2
+
+
+def test_add_copy_display_places():
```

Comment:
Also, it will be nice to use `pytest.mark.parametrize` as these 3 tests have similar structure